lib/wordAnalysis.js

```diff
@@ -0,0 +1,43 @@
+
+function wordStats(arrayOfWords) {
+  let counts = {};
+  for (let word of arrayOfWords) {
+    if (!counts[word]) {
+      counts[word] = 1;
+    } else {
+      counts[word] += 1;
+    }
+  }
+  let greatestFrequencies = Object.values(counts).sort().reverse().slice(0, 5);
+
+  let mostCommon = {};
+  for (let key of Object.keys(counts)) {
+    if (greatestFrequencies.includes(counts[key])) {
```

Comment:
This won't get the top 5. If the top 5 all have the same counts, it'll get one of the top 5 and then start at #6 (for example).